<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MianWindowClass</class>
 <widget class="QMainWindow" name="MianWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园行走最优路径查询系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="MainWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="MapView" name="mapWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,3,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="userWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,1,1">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="buildingsInfoWidget" native="true">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>ID:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="buildingsInfoID">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>名称：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="buildingsInfoName">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="addBuildingsToListPushButton">
                  <property name="text">
                   <string>加入列表</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>353</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListWidget" name="buildingsListWidget">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>327</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QPushButton" name="buildingsListClearPushButton">
                    <property name="text">
                     <string>清空</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="buildingsListQueryPushButton">
                    <property name="text">
                     <string>查询</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>759</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QWidget" name="addWayPointWidget" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(214, 214, 214);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1,1,1,2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>坐标：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>X:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="XDoubleSpinBox">
                     <property name="maximum">
                      <double>9999.989999999999782</double>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Y:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="YDoubleSpinBox">
                     <property name="maximum">
                      <double>9999.989999999999782</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>名称:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="addLandMarkName"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>类型:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="wayPointComboBox">
                   <item>
                    <property name="text">
                     <string>建筑物</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>道路（拐弯或分叉口）</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="cancelPushButton">
                   <property name="text">
                    <string>取消</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="confirmPushButton">
                   <property name="text">
                    <string>确认</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <item>
                       <widget class="QLabel" name="label_6">
                        <property name="text">
                         <string>路径点ID1</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSpinBox" name="wayPointID1spinBox"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <widget class="QLabel" name="label_7">
                        <property name="text">
                         <string>路径点ID2</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSpinBox" name="wayPointID2spinBox"/>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QPushButton" name="addRoadCancelPushButton">
                      <property name="text">
                       <string>取消</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="addRoadConfirmPushButton">
                      <property name="text">
                       <string>确定</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_15">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_13">
                      <item>
                       <widget class="QLabel" name="label_10">
                        <property name="text">
                         <string>路径点Id:</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSpinBox" name="wayPointIdSpinBox"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_14">
                      <item>
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QPushButton" name="deletePushButton">
                        <property name="text">
                         <string>删除路径点</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QPushButton" name="exportWayPointPushButton">
                      <property name="text">
                       <string>导出路径点</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="exportRoadPushButton">
                      <property name="text">
                       <string>导出道路</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapView</class>
   <extends>QWidget</extends>
   <header>MapView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MianWindow.qrc"/>
 </resources>
 <connections/>
</ui>
